doc/classes/World3D.xml

```diff
@@ -13,6 +13,9 @@
 		<member name="camera_attributes" type="CameraAttributes" setter="set_camera_attributes" getter="get_camera_attributes">
 			The default [CameraAttributes] resource to use if none set on the [Camera3D].
 		</member>
+		<member name="compositor" type="Compositor" setter="set_compositor" getter="get_compositor">
+			The default [Compositor] resource to be use if none is set on the [Camera3D]. Will be replaced by the first [WorldEnvironment] registered to this World3D.
```

Comment:
```suggestion
			The default [Compositor] resource to use if none is set on the [Camera3D]. Will be replaced by the first [WorldEnvironment] registered to this World3D.
```
My bad mixed up the sentence when fixing the grammar of the one above

Reply:
oh yeah i read "to be use**d**" and it looked fine :rofl: